cwms-data-api/src/test/java/cwms/cda/api/VersionedTimeseriesControllerTestIT.java

```diff
@@ -29,13 +30,11 @@
 @Tag("integration")
 public class VersionedTimeseriesControllerTestIT extends DataApiTestIT {
 
-    private static final String OFFICE_STR = "SPK";
-    private static TestAccounts.KeyUser user = TestAccounts.KeyUser.SPK_NORMAL;
+    private static final String OFFICE_STR = "SWT";
     private static final String locationId = "TsVersionedTestLoc";
     private static final String tsId = locationId + ".Flow.Inst.1Hour.0.raw";
     private static final String BEGIN_STR = "2008-05-01T15:00:00Z";
     private static final String END_STR = "2008-05-01T17:00:00Z";
-    public static final String resourcePath = String.format("/cwms/cda/api/%s/", OFFICE_STR.toLowerCase());
```

Comment:
This seems like a good idea, i suspect you may want to pull the latest changes into your fork and rebase.

Reply:
Yes, I needed to rebase on develop.  I wasn't intentionally trying to delete that line.